torch/distributed/tensor/parallel/api.py

```diff
@@ -21,8 +22,8 @@
 
 def parallelize_module(  # type: ignore[return]
     module: nn.Module,
-    device_mesh: DeviceMesh,
-    parallelize_plan: Union[ParallelStyle, Dict[str, ParallelStyle]],
+    device_mesh: Optional[DeviceMesh] = None,
+    parallelize_plan: Optional[Union[ParallelStyle, Dict[str, ParallelStyle]]] = None,
```

Comment:
maybe explain why you want this?
why not just skip calling `parallelize_module` at all?

Reply:
You are so fast in reviewing it, even before I finish writing the PR desc. 
Now the motivation is added :)